client/src/components/RequestModal.js

```diff
@@ -101,11 +101,26 @@ export default class RequestModal extends Component {
 
     submitRequest(e) {
         e.preventDefault()
+        
         this.setState({
             submitting: true
         }, async () => {
             try {
-                const result = true;
+                const jwtVal = document.cookie.replace(/(?:(?:^|.*;\s*)jwt\s*\=\s*([^;]*).*$)|^.*$/, "$1");
+                const parsedJWT = JSON.parse(atob(jwtVal.split('.')[1]));
+                const requesterId = parsedJWT.id
+                let timeOffset = new Date().getTimezoneOffset()
```

Comment:
This should probably be made a state, but it would only really make sense to do this when we have a timezone dropdown